### DISCUSSION TITLE
作者大大  装备推荐跟符文推荐有计划加一个自动配置吗

### DISCUSSION THREAD

**Discussion by yooo23** [LeagueAkari/LeagueAkari#161] [Ideas]
作者大大  作者大大  装备推荐跟符文推荐有计划加一个自动配置吗吗

**Comment by yooo23** [comment]
感觉软件的自动化很好用要是有符文自动配置就太好了

**Comment by Hanxven** [comment]
之后的版本 (v1.3.3 或之后) 将尝试实现一个具有基础功能的符文配置器。

不和 OP.GG 绑定。
